### ISSUE TITLE
Improve display of mapping relations expressed with SKOS

### ISSUE THREAD

**Issue by rettinghaus** [hbz/lobid-gnd#312]
It would be nice if the MACS-Mapping cross concordance information could be integrated/displayed somehow in the HTML view. 
Maybe just add the corresponding links in the `Siehe auch` row?

Example: https://d-nb.info/gnd/4154647-7
See also: https://wiki.dnb.de/display/GND/GND-Mappings+zu+externen+Thesauri

**Comment by rettinghaus** [comment]
Any plans for this?

**Comment by rettinghaus** [comment]
I just noticed, that the links now are present, but marked as "closeMatch". 

<img width="706" alt="Image" src="https://github.com/user-attachments/assets/a0898fb3-7183-4c5e-ab30-0488415dc074" />

Maybe a better German title could be found.

**Comment by acka47** [comment]
> I just noticed, that the links now are present, but marked as "closeMatch".

Yes, they are as `skos:closeMatch` in the [underlying JSON-LD]():

```json
{
   "closeMatch":[
      {
         "id":"https://data.bnf.fr/ark:/12148/cb119712096",
         "label":"https://data.bnf.fr/ark:/12148/cb119712096"
      },
      {
         "id":"https://data.bnf.fr/ark:/12148/cb16269444g",
         "label":"https://data.bnf.fr/ark:/12148/cb16269444g"
      },
      {
         "id":"http://id.loc.gov/authorities/subjects/sh85049452",
         "label":"http://id.loc.gov/authorities/subjects/sh85049452"
      },
      {
         "id":"http://purl.org/bncf/tid/1712",
         "label":"http://purl.org/bncf/tid/1712"
      }
   ]
}
``` 

> Maybe a better German title could be found.

You are right, this is not ideal. AFAIK, currently, we do not have any configuration option for the labels but only use the labels from [GND Ontology](https://d-nb.info/standards/elementset/gnd). As `closeMatch` comes from SKOS and not GNDO it does not get a decent human-readable label. I will rename the issue and we will look into it.

**Comment by acka47** [comment]
Currently, all SKOS mapping relations are used in GND data:

- [`skos:closeMatch`](https://lobid.org/gnd/search?q=_exists_%3AcloseMatch)
- [`skos:exactMatch`](https://lobid.org/gnd/search?q=_exists_%3AexactMatch)
- [`skos:broadMatch`](https://lobid.org/gnd/search?q=_exists_%3AbroadMatch)
- [`skos:narrowMatch`](https://lobid.org/gnd/search?q=_exists_%3AnarrowMatch)
- [`skos:relatedMatch`](https://lobid.org/gnd/search?q=_exists_%3ArelatedMatch)

In the UI:

- `closeMatch` is displayed as "closeMatch"
- `exactMatch` is displayed as "Entspricht"
- `broadMatch` is displayed as "Oberbegriff"
- `narrowMatch` is displayed as "narrowMatch"
-  `relatedMatch` is displayed as "Verwandet Begriff"

Looking at `exactMatch`, `broadMatch` and `relatedMatch`, my last comment apparently wasn't correct and we already have a way to configure human-readable labels. I will look into this and how we can also cover `closeMatch` and `narrowMatch`.

**Comment by rettinghaus** [comment]
done 😃
